packages/massa-proto-cli/src/protobuf.ts

```diff
@@ -57,7 +57,7 @@ export async function getProtoFunction(protoPath: string): Promise<ProtoFile> {
   // get the arguments of the Helper
   const argFields: FunctionArguments[] = Object.entries(helper.fields)
     .filter(([, value]) => value)
-    .map(([name, field]) => ({ name, type: field.type }));
+    .map(([name, field]) => ({ name, type: field['type'] }));
```

Comment:
i'm curious, why ?

Reply:
the compiler was angry. we don't know exactly why.
Then we updated it like this:

`.map(([name, field]) => ({
      name,
      type: (field as { type: string; id: number }).type,
    }));`